Core/src/org/sleuthkit/autopsy/textextractors/TikaTextExtractor.java

```diff
@@ -110,7 +120,7 @@
 
     private static final java.util.logging.Logger tikaLogger = java.util.logging.Logger.getLogger("Tika"); //NON-NLS
 
-    private final ExecutorService tikaParseExecutor = Executors.newSingleThreadExecutor();
+    private final ExecutorService executorService = Executors.newSingleThreadExecutor();
```

Comment:
We should pass a ThreadFactory to this executor to give the threads meaningful names.